src/test/java/FramesTest.java

```diff
@@ -0,0 +1,41 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.TimeUnit;
+
+
+public class FramesTest {
+
+    private WebDriver driver;
+    private String URL = "https://the-internet.herokuapp.com/iframe";
+    private final String TEXT = "Hello world!";
+
+    @BeforeClass
+    public void setUp() {
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(6000, TimeUnit.SECONDS);
```

Comment:
100 hours =)))) Make 10 seconds, if you have questions about implicit waiter - you can ask and I will explain how it works